src/Driver/RedisDriver.php

```diff
@@ -32,8 +32,19 @@ public function enqueue($queue, $command)
      */
     public function dequeue($queue)
     {
-        list($_, $message) = $this->redis->blPop($queue, 5) ?: null;
+        list($_, $message) = $this->redis->blPop($queue, static::TIMEOUT) ?: null;
```

Comment:
Both ways are ugly and I wish there was a better way to do it, but I can drop the `$_` as it just pollutes.